public/index.html

```diff
@@ -30,6 +31,8 @@
   <script src="app/assetlibrary/addbookmarklet/assetLibraryAddBookmarkletController.js"></script>
   <script src="app/assetlibrary/addlink/assetLibraryAddLinkFactory.js"></script>
   <script src="app/assetlibrary/addlink/assetLibraryAddLinkController.js"></script>
+  <script src="app/assetlibrary/upload/assetLibraryUploadFileFactory.js"></script>
```

Comment:
Can you add this after `<script src="app/assetlibrary/list/assetLibraryListController.js"></script>` ?